docs/kyma/04-01-overview.md

```diff
@@ -13,7 +13,8 @@ To make the local development process easier, we introduced the **Kyma Lite** co
 | `cms` | ✅ | ✅ |
 | `cluster-essentials` | ✅ | ✅ |
 | `application-connector` | ✅ | ✅ |
-| `ark` | ✅ | ⛔️ |
+| `velero-essentials` | ✅ | ⛔️ |
```

Comment:
It is because we have to install a CRD first and then the CR. This sequence can be achieved using helm hooks like `crd-install` but I couldn't as the tiller version in Kyma doesn't support it.
Please refer https://github.com/kyma-project/kyma/issues/3887

Reply:
ok, I agree till helm is supporting it